google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -27,9 +27,15 @@
 from google.cloud import documentai_toolbox
 
 from google.cloud.documentai_toolbox import constants
-from google.cloud.documentai_toolbox.wrappers.page import Page
+from google.cloud.documentai_toolbox.wrappers.page import Page as toolbox_page
```

Comment:
Should `toolbox_page` be Capitalized to make it clear that it's a class? E.g. `ToolboxPage`

Reply:
Done.